### PR TITLE
Fix no credential message

### LINKED ISSUE #110: Not having the required credentials is a blocker
After accessing the Data Vault and identifying that there are no credentials for the required fields we can prompt the user to a verification service.

We can have a whitelist with, initially, [VC Issuer](https://github.com/rsksmart/email-vc-issuer) url and its address (to compare against the signer of the disclosed credential).

### PR DESCRIPTION
**Pull request by diegograsso**
Closes #110

### CODE REVIEW

**Review by jessgusclark — DISMISSED** [review]
Works well. A few thoughts:

1. The link text says "identity manager" but it links to the "issuer". I'm currently working on adding the requests to the manager, so perhaps we should link them there?
2. If I have a credential but not a name declarative detail it does not prompt anything, even though in the backend it says the declarative detail is required. Should this also be accounted for? Perhaps linking to the manager would work for both.
3. Let's add a Cypress test for this. You can mock a response from the datavault. Take a look [at this file](https://github.com/rsksmart/rlogin-sample-apps/blob/main/permissioned-app/frontend-app/cypress/integration/permissioned.spec.js#L33-L37) and the [fixtures here](https://github.com/rsksmart/rlogin-sample-apps/tree/main/permissioned-app/frontend-app/cypress/fixtures)

File: sample/front/index.html [hunk 1]
```diff
@@ -94,6 +94,7 @@ <h2>Actions</h2>
         dataVaultOptions: {
           package:RIFDataVault,
           serviceUrl: 'https://data-vault.identity.rifos.org',
+          appUrl: 'https://identity.rifos.org',
```
Comment (ilanolkies) [inline sample/front/index.html:97]: We can make this not configurable.

**Review by ilanolkies — CHANGES_REQUESTED** [review]
Can we make only the word "RIF Identity Manager" a link? With uppers and "RIF"

Screen Shot 2021-08-17 at 17 52 20

**Review by ilanolkies — APPROVED** [review]
Screen Shot 2021-08-17 at 19 27 48

LRGTM!

**Pull request merged**